README.md

```diff
@@ -308,6 +309,60 @@ The backend system would collect all `sensu_check` resources.
   Sensu_check <<||>>
 ```
 
+### Hiera resources
+
+All the types provided by this module can have their resources defined via Hiera. A type such as `sensu_check` would be defined via `sensu::backend::checks`.
+
+The following example adds a check via Hiera:
```

Comment:
This should be rephrased as the example does a lot more then adding a check

Reply:
Updated wording.